c7n/resources/glue.py

```diff
@@ -412,3 +412,66 @@ def process(self, resources):
                 client.delete_workflow(Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+
+@resources.register('glue-catalog')
+class GlueDataCatalog(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_data_catalog_encryption_settings', 'DataCatalogEncryptionSettings', None)
+        arn_type = 'catalog'
+        id = name = 'CatalogId'
+
+    permissions = ('glue:GetDataCatalogEncryptionSettings',)
+
+    def augment(self, resources):
+        resource = []
+        resource.append(resources)
+        return resource
+
+
+@GlueDataCatalog.action_registry.register('set-glue-catalog-encryption')
```

Comment:
you can remove the redundant glue-catalog in the name since its on the glue catalog resource.